guides/Makefile

```
@@ -19,13 +19,11 @@ clean:
 		$(MAKE) -s --directory="$$DIR" clean ; \
 	done
 
-linkchecker:
-	echo "do not check links"
-	# find build -type f -name index\*html | xargs linkchecker -r1 -f common/linkchecker.ini --check-extern
+linkchecker:	
+	find build -type f -name index\*html | xargs linkchecker -r1 -f common/linkchecker.ini --check-extern
```

https://linkchecker.github.io/linkchecker/man/linkchecker.html has `linkchecker --ignore-url=/secret` as an example. I wonder if we can somehow use that. Like `--ignore-url=https://access.redhat.com/documentation/en-us/red_hat_satellite/`. What I tried:
```suggestion
	find build -type f -name index\*html | xargs linkchecker --ignore-url=${IGNORE_RH_SAT_DOCS} -r1 -f common/linkchecker.ini --check-extern
```

And at line 3 `IGNORE_RH_SAT_DOCS = https://access.redhat.com/documentation/en-us/red_hat_satellite/`. That made the build pass for me. It could even be more specific and only exclude `https://access.redhat.com/documentation/en-us/red_hat_satellite/6.10/html-single/managing_configurations_using_puppet_integration/index` of course.

Then we should remember to remove that in branches that do map to Satellite releases though.

---

For the linkchecker example, it is probably for a permanent issue.  Since we won't use this permanently, it will require enabling and disabling, so I see little value in figuring this out.

---

Oh, we already have `guides/common/linkchecker.ini` with a list of things to ignore. The easiest would be to add `access.redhat.com/documentation/en-us/red_hat_satellite/6.10/html-single/managing_configurations_using_puppet_integration/index` there. That should be also added to the "adding a new guide"-guide because this is something that will keep coming back.

---

I said in my comment below that there will be a wave of various things that will break the Satellite build in the coming weeks. 
We can either make one change here, I open a ticket and assign to RH docs to readd on GA. Or we could make several changes.
 I don't see the value. Am happy to close this PR and you can do what you see fit.

---

Sounds good to me: I opened https://github.com/theforeman/foreman-documentation/pull/1063 as an alternative.

---

Great that you have so much time for all these things. Thank you!

---

And thanks for fixing the (un)related broken links. I think in the past I've noticed a few others where there was no "label" (is it called that?). I mean the part in `[]`.